src/components/MovieBooking/MovieModal.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
```

Comment:
Modal is not centrally aligned. And background must be blur when modal is opened.